src/index.js

```diff
@@ -1,4 +1,82 @@
 import './style.scss';
 
 // You can specify which plugins you need
-// import { Tooltip, Toast, Popover } from 'bootstrap';
\ No newline at end of file
+// import { Tooltip, Toast, Popover } from 'bootstrap';`
+
+const leaderBoardApi = document.querySelector('#leaderboard-api');
+
+let gameID = 0;
+async function createGameId() {
+  const data = {
+    name: 'leader board',
+  };
+  let response = await fetch(`${leaderBoardApi.action}games/`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json', // 'Content-Type': 'application/x-www-form-urlencoded',
+    },
+    body: JSON.stringify(data),
+  });
+  response = await response.json(); // .then(reply => reply.json())
+  // .then(json =>
+
+  const { result } = response;
+  const indexColumn = result.indexOf(':');
+  const indexAdded = result.indexOf('added');
+  console.log(result);
+  gameID = result.substring(indexColumn + 2, indexAdded - 1);
+  console.log(gameID);
```

Comment:
- [ ] Console logs kill performance and could expose confidential data in more sophisticated apps, hence it is not good practice to commit and push them. Kindly remove them here.